websocketserver.go

```diff
@@ -78,12 +78,11 @@ func (s *WebsocketServer) ListenAndServe(address string) (io.Closer, error) {
 		s.log.Print(err)
 		return nil, err
 	}
-	addr := l.(*net.TCPListener).Addr()
 
 	// Run service on configured port.
 	server := &http.Server{
 		Handler: s,
-		Addr:    addr.String(),
+		Addr:    l.(*net.TCPListener).Addr().String(),
```

Comment:
Do you still need the type assertion?

same below.

Reply:
Yes, because `net.Listener` does not have an `Addr()` method.

Reply:
Am I looking at something wrong?

https://golang.org/pkg/net/#Listener

Reply:
No. My bad, do not know what I was looking at.